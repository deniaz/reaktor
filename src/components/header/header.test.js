import React from 'react';
import { create } from 'react-test-renderer';

import { Header } from '.';

describe('Component', () => {
  describe('Header', () => {
    it('matches snapshot', () => {
      const component = create(<Header />);
      const tree = component.toJSON();

      expect(tree).toMatchSnapshot();
    });
  });
});
