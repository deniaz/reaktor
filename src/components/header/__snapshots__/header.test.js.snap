// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Header matches snapshot 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <a
      className="logo-area"
      href="#"
    >
      <img
        alt="MyConf"
        className="logo"
        src="test-file-stub"
      />
    </a>
    <nav
      className="navigation navigation"
    >
      <ul>
        <li
          className="list-item"
        >
          <a
            className="navigation-link"
            href="#"
          >
            Speakers
          </a>
        </li>
        <li
          className="list-item"
        >
          <a
            className="navigation-link"
            href="#"
          >
            Workshops
          </a>
        </li>
        <li
          className="list-item"
        >
          <a
            className="navigation-link"
            href="#"
          >
            Venue
          </a>
        </li>
        <li
          className="list-item"
        >
          <a
            className="navigation-link"
            href="#"
          >
            Sponsors
          </a>
        </li>
        <li
          className="list-item"
        >
          <a
            className="navigation-link"
            href="#"
          >
            About
          </a>
        </li>
        <li
          className="list-item"
        >
          <button
            className="button button--primary"
            disabled={false}
            onClick={[Function]}
          >
            Tickets
          </button>
        </li>
      </ul>
    </nav>
  </div>
</header>
`;
